plutus-playground-client/src/AjaxUtils.purs

```diff
@@ -110,7 +111,8 @@ showErrorDescription (ConnectionError err) = text $ "ConnectionError: " <> err
 
 ajaxErrorPane :: forall p i. AjaxError -> HTML p i
 ajaxErrorPane error =
-  div_
+  div
+    [ class_ $ ClassName "ajax-error" ]
```

Comment:
it feels like we should have a module for all `ClassName`s that aren't bootstrap or somewhere else.

Reply:
I see what you mean, but I'm not convinced that would work well. You'd end up with a load of strings that bore no real relation to each other bar their type. Each one would only be imported into a single other module. There'd be no real reuse and no real grouping by-use.

It _would_ make it easier to ensure that `id` tags were unique.

That said, I think there is an argument for a) pulling together all the class names that were genuinely used in more than one place, and b) having a rule of thumb that says classname definitions should be top-level, to make them more visible within their module.

What do you think?

Reply:
well I definitely agree with a) and I think that was half my point here. `ajax-error` in particular seems pretty generic? I'd never thought of b) but yes I think it's a great idea.

Reply:
Actually `ajax-error` only gets used inside `ajaxErrorPane` so... 🤷‍♂️ 

In that case I suggest we adopt b) from here on in, and I'll look at resolving a) in  another commit.

Reply:
agreed